common/sw-update/dev-updates-profile.cpp

```diff
@@ -27,55 +27,54 @@ namespace rs2
             _update_profile.serial_number = serial;
         }
 
-        bool dev_updates_profile::retrieve_updates(versions_db_manager::component_part_type comp, std::pair<sw_update::version, dev_updates_profile::update_description> &recommended_version)
+        bool dev_updates_profile::retrieve_updates(component_part_type comp)
         {
-            bool update_required(false);
+            bool update_available(false);
 
             if (_update_profile.device_name.find("Recovery") == std::string::npos)
             {
-                std::map<version, update_description> &versions_vec((comp == versions_db_manager::FIRMWARE) ?
+                std::map<version, update_info>& versions_vec((comp == FIRMWARE) ?
```

Comment:
I suggest `version_info` since it's information about a version

Reply:
👍